java/api/answer/ConceptRow.java

```diff
@@ -55,6 +56,17 @@ public interface ConceptRow {
     @CheckReturnValue
     QueryType getQueryType();
 
+    /**
+     * Retrieves the executed query's structure of this <code>ConceptRow</code>.
+     * Shared between all the rows in a QueryAnswer.
```

Comment:
Maybe add a note to these docs throughout "When enabled as a query option, retrieves the executed query structure of this ConceptRow" or something like that